cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -277,10 +257,17 @@ int main(int argc, char** argv)
                 }
             }
 
-            // After MSA is generated for batch, reset batch to make room for next set of POA groups.
+            // After MSA/consensus is generated for batch, reset batch to make room for next set of POA groups.
             batch->reset();
 
-            std::cout << "Processed windows " << window_count << " - " << i << std::endl;
+            if (status == StatusType::success)
```

Comment:
Could you elaborate a bit on this change? Not sure I'm quite following this

Reply:
in case than number of windows is more than the capacity available on GPU, the for loop breaks into smaller number of windows. If adding window `i` in `batch->add_poa_group` turns out to exceed the limit, it wont be processed, therefore we print `i-1` to account for the fact that window `i` was excluded at that round. 
I can add this explanation as a comment to make this part of code more clear.
```
            if (status == StatusType::success)
            {
                std::cout << "Processed windows " << window_count << " - " << i << std::endl;
            }
            else
            {
                std::cout << "Processed windows " << window_count << " - " << i - 1 << std::endl;
            }
```

Reply:
added comment to clarify.